docs/en/03_How_tos/external_http_requests_with_proxy.md

```diff
@@ -20,6 +20,11 @@ please add the following IP addresses:
 
 You can look up the implementation details in the `CwpInitialisationFilter` in the `cwp-core` module.
 
+## Known limitations
+
+Requests to internally resolving cwp hostnames will fail to connect when using the proxy
+Additionally requests over https (i.e. https://yourstack-uat.cwp.govt.nz)  will fail to connect to the webserver via the cwp proxy and will fail without using the proxy too. We reccomend adding a custom domain to the stack via the [CWP Service Desk](https://www.cwp.govt.nz/service-desk) and using that instead (i.e. yourstack-test.govt.nz) so that the request is resolved via external DNS.
```

Comment:
```suggestion
Additionally requests over https (i.e. https://yourstack-uat.cwp.govt.nz) will fail to connect to the webserver via the CWP proxy and will fail without using the proxy too. We recommend adding a custom domain to the stack via the [CWP Service Desk](https://www.cwp.govt.nz/service-desk) and using that instead (i.e. yourstack-test.govt.nz). This will ensure that the request is resolved via external DNS.
```

>will fail to connect to the webserver via the CWP proxy and will fail without using the proxy too

If it fails in both cases, I don't think it's worth including this sentence

Reply:
Just this from me outstanding: "If it fails in both cases, I don't think it's worth including this sentence"

Reply:
I think in context of the previous line it makes sense.